com.microsoft.mrtk.input/Assets/Input Actions/MRTK Spatial Mouse Input Actions.inputactions

```diff
@@ -0,0 +1,94 @@
+{
+    "name": "MRTK Spatial Mouse Input Actions",
+    "maps": [
```

Comment:
Could this map be included in our default input actions asset? We have several maps in it already (I think HMD, left hand, right hand, and eye gaze), so this _could_ logically fit. Or is there a benefit to it being split out?

Reply:
I was intentionally making this separate because Spatial Mouse is an optional controller and I didn't want to add + activate Input Actions that weren't being used. @Zee2 What do you think?

Reply:
I'm fine with either, both justifications are totally reasonable. I think including it in the default action asset isn't a big cost, though, because the user will never really need to worry about it or do anything about it if they don't care about mouse input...